File: fingertip/src/main/java/com/praszapps/fingertip/model/di/module/ActivityBuilder.java
package com.praszapps.fingertip.model.di.module;

import com.praszapps.fingertip.view.FingertipActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class ActivityBuilder {

    @ContributesAndroidInjector(modules = FingertipActivityModule.class)
    abstract FingertipActivity bindFingerprintActivity();

}

File: fingertip/src/main/java/com/praszapps/fingertip/model/di/module/ActivityBuilder.kt
package com.praszapps.fingertip.model.di.module

import com.praszapps.fingertip.view.FingertipActivity

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class ActivityBuilder {

    @ContributesAndroidInjector(modules = arrayOf(FingertipActivityModule::class))
    internal abstract fun bindFingerprintActivity(): FingertipActivity

}
